diff --git a/dpdk/config/defconfig_x86_64-native-linuxmodules-gcc b/dpdk/config/defconfig_x86_64-native-linuxmodules-gcc
new file mode 100644
--- /dev/null
+++ b/dpdk/config/defconfig_x86_64-native-linuxmodules-gcc
@@ -0,0 +1,103 @@
+#
+# Minimal Configuration for igb_uio and rte_kni Kernel Modules
+#
+# Copyright (c) 2014 Semihalf.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS'' AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+# OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+# SUCH DAMAGE.
+
+#include "common_linuxapp"
+
+## machine can define specific variables or action for a specific board
+CONFIG_RTE_MACHINE="native"
+
+## define the architecture we compile for.
+CONFIG_RTE_ARCH="x86_64"
+CONFIG_RTE_ARCH_X86_64=y
+
+## The compiler we use.
+CONFIG_RTE_TOOLCHAIN="gcc"
+CONFIG_RTE_TOOLCHAIN_GCC=y
+
+# Compile the argument parser library
+CONFIG_RTE_LIBRTE_KVARGS=n
+# Compile burst-oriented IGB & EM PMD drivers
+CONFIG_RTE_LIBRTE_EM_PMD=n
+CONFIG_RTE_LIBRTE_IGB_PMD=n
+# Compile burst-oriented IXGBE PMD driver
+CONFIG_RTE_LIBRTE_IXGBE_PMD=n
+# Compile burst-oriented I40E PMD driver
+CONFIG_RTE_LIBRTE_I40E_PMD=n
+# Compile burst-oriented FM10K PMD
+CONFIG_RTE_LIBRTE_FM10K_PMD=n
+# Compile burst-oriented Cisco ENIC PMD driver
+CONFIG_RTE_LIBRTE_ENIC_PMD=n
+# Compile burst-oriented VIRTIO PMD driver
+CONFIG_RTE_LIBRTE_VIRTIO_PMD=n
+# Compile burst-oriented VMXNET3 PMD driver
+CONFIG_RTE_LIBRTE_VMXNET3_PMD=n
+# Compile example software rings based PMD
+CONFIG_RTE_LIBRTE_PMD_RING=n
+# Compile link bonding PMD library
+CONFIG_RTE_LIBRTE_PMD_BOND=n
+# Compile software PMD backed by AF_PACKET sockets (Linux only)
+CONFIG_RTE_LIBRTE_PMD_AF_PACKET=n
+# Compile null PMD
+CONFIG_RTE_LIBRTE_PMD_NULL=n
+# Compile librte_timer
+CONFIG_RTE_LIBRTE_TIMER=n
+# Compile librte_cfgfile
+CONFIG_RTE_LIBRTE_CFGFILE=n
+# Compile librte_cmdline
+CONFIG_RTE_LIBRTE_CMDLINE=n
+# Compile librte_hash
+CONFIG_RTE_LIBRTE_HASH=n
+# Compile librte_jobstats
+CONFIG_RTE_LIBRTE_JOBSTATS=n
+# Compile librte_lpm
+CONFIG_RTE_LIBRTE_LPM=n
+# Compile librte_acl
+CONFIG_RTE_LIBRTE_ACL=n
+# Compile librte_power
+CONFIG_RTE_LIBRTE_POWER=n
+# Compile librte_net
+CONFIG_RTE_LIBRTE_NET=n
+# Compile librte_ip_frag
+CONFIG_RTE_LIBRTE_IP_FRAG=n
+# Compile librte_meter
+CONFIG_RTE_LIBRTE_METER=n
+# Compile librte_sched
+CONFIG_RTE_LIBRTE_SCHED=n
+# Compile the distributor library
+CONFIG_RTE_LIBRTE_DISTRIBUTOR=n
+# Compile the reorder library
+CONFIG_RTE_LIBRTE_REORDER=n
+# Compile librte_port
+CONFIG_RTE_LIBRTE_PORT=n
+# Compile librte_table
+CONFIG_RTE_LIBRTE_TABLE=n
+# Compile librte_pipeline
+CONFIG_RTE_LIBRTE_PIPELINE=n
+# Compile the test application
+CONFIG_RTE_APP_TEST=n
+# Compile the PMD test application
+CONFIG_RTE_TEST_PMD=n
