--- ifup-eth	2013-01-04 06:08:15.590169778 -0800
+++ ifup-eth.contrail	2013-01-04 04:01:45.583587121 -0800
@@ -252,10 +252,11 @@
 
         if ! LC_ALL=C ip addr ls ${REALDEVICE} | LC_ALL=C grep -q "${ipaddr[$idx]}/${prefix[$idx]}" ; then
             [ "${REALDEVICE}" != "lo" ] && \
-            if ! /sbin/arping -q -c 2 -w 3 -D -I ${REALDEVICE} ${ipaddr[$idx]} ; then
-                net_log $"Error, some other host already uses address ${ipaddr[$idx]}."
-                exit 1
-            fi
+            # contrail workaround
+            #if ! /sbin/arping -q -c 2 -w 3 -D -I ${REALDEVICE} ${ipaddr[$idx]} ; then
+            #    net_log $"Error, some other host already uses address ${ipaddr[$idx]}."
+            #    exit 1
+            #fi
 
             if ! ip addr add ${ipaddr[$idx]}/${prefix[$idx]} \
               brd ${broadcast[$idx]:-+} dev ${REALDEVICE} ${SCOPE} label ${DEVICE}; then
@@ -275,6 +276,8 @@
         fi
     done
 
+    # contrail workaround
+    sleep 2
     # Set a default route.
     if [ "${DEFROUTE}" != "no" ] && [ -z "${GATEWAYDEV}" -o "${GATEWAYDEV}" = "${REALDEVICE}" ]; then
 	# set up default gateway. replace if one already exists
@@ -288,6 +291,9 @@
 		${SRC} ${WINDOW:+window $WINDOW} dev ${REALDEVICE} ||
 			net_log $"Erorr adding default gateway for ${REALDEVICE}."
 	fi
+	# contrail workaround
+	sleep 2
+	ping -q -c 5 ${GATEWAY} >/dev/null
     fi
     fi
 fi
