Index: nova/tools/pip-requires
===================================================================
--- nova.orig/tools/pip-requires	2013-03-13 17:47:30.180806431 -0700
+++ nova/tools/pip-requires	2013-03-13 17:47:45.816807224 -0700
@@ -20,7 +20,6 @@
 Babel>=0.9.6
 iso8601>=0.1.4
 httplib2
-setuptools_git>=0.4
 python-cinderclient>=1.0.1
 python-quantumclient>=2.2.0,<3.0.0
 python-glanceclient>=0.5.0,<2
