Index: nova/tools/hacking.py
===================================================================
--- nova.orig/tools/hacking.py	2013-03-13 17:47:30.280806436 -0700
+++ nova/tools/hacking.py	2013-03-13 17:47:36.364806745 -0700
@@ -646,7 +646,7 @@
     #include nova path
     sys.path.append(os.getcwd())
     #Run once tests (not per line)
-    once_error = once_git_check_commit_title()
+    #once_error = once_git_check_commit_title()
     #NOVA error codes start with an N
     pep8.SELFTEST_REGEX = re.compile(r'(Okay|[EWN]\d{3}):\s(.*)')
     pep8.ERRORCODE_REGEX = re.compile(r'[EWN]\d{3}')
