Index: nova/plugins/xenserver/xenapi/etc/xapi.d/plugins/xenhost
===================================================================
--- nova.orig/plugins/xenserver/xenapi/etc/xapi.d/plugins/xenhost	2013-03-13 17:47:30.328806438 -0700
+++ nova/plugins/xenserver/xenapi/etc/xapi.d/plugins/xenhost	2013-03-13 17:47:31.088806477 -0700
@@ -41,7 +41,7 @@
 pluginlib.configure_logging("xenhost")
 
 host_data_pattern = re.compile(r"\s*(\S+) \([^\)]+\) *: ?(.*)")
-config_file_path = "/usr/etc/xenhost.conf"
+config_file_path = "/etc/xcp/xenhost.conf"
 DEFAULT_TRIES = 23
 DEFAULT_SLEEP = 10
 
