diff --git a/setup.py b/setup.py
index 403e32c..38f5590 100644
--- a/setup.py
+++ b/setup.py
@@ -43,7 +43,7 @@ setuptools.setup(
           ],
       cmdclass=common_setup.get_cmdclass(),
       packages=setuptools.find_packages(exclude=['bin', 'smoketests']),
-      install_requires=requires,
+      install_requires=[],
       dependency_links=depend_links,
       include_package_data=True,
       test_suite='nose.collector',
